internal/dic/dic.go

```diff
@@ -136,53 +136,53 @@ func LoadSimple(path string) (d *Dic, err error) {
 	return load(&r.Reader, false)
 }
 
-func load(r *zip.Reader, full bool) (d *Dic, err error) {
-	d = new(Dic)
+func load(r *zip.Reader, full bool) (*Dic, error) {
```

Comment:
__[Go Meta Linter]__ [gocyclo] cyclomatic complexity 20 of function load() is high (> 15) ([link](https://sider.review/gh/repos/21228174/pulls/153#issue-4493898))

<sub>posted by Sider</sub>